core/domain/domain.core.yml

```diff
@@ -25,6 +25,8 @@ intents:
   - symptom_free
   - send_validation_code:
       triggers: action_send_validation_code
+  - external_send_checkin:
```

Comment:
I choose to avoid the external prefix which I found ugly, I we really want it, it should also be added to the `send_validation_code` intent which is also external.

Reply:
I will remove it